data/brands/shop/shoes.json

```diff
@@ -874,6 +874,17 @@
         "shop": "shoes"
       }
     },
+    {
+      "displayName": "Pataugas",
+      "id": "q3368657-c57706",
+      "locationSet": {"include": ["fr"]},
+      "tags": {
+        "brand": "Pataugas",
+        "brand:wikidata": "Q3368657",
+        "name": "Q3368657",
```

Comment:
Wrong name

Reply:
oups ! You are right sorry. I just pushed a fix.